scripts/chairman.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+genesis="33873"
+genesis_root="configuration/${genesis}"
+genesis_file="${genesis_root}/genesis.json"
+if test ! -f "${genesis_file}"
+then echo "ERROR: genesis ${genesis_file} does not exist!">&1; exit 1; fi
+
+cabal new-build "exe:cardano-cli"
+genesis_hash="$(cabal new-exec -- cardano-cli real-pbft print-genesis-hash --genesis-json ${genesis_file})"
+
+CMD=`find dist-newstyle/ -type f -name "cardano-node"`
+test -n "${CMD}" || {
```

Comment:
I think that instead of `new-build` & `new-exec` one can just use `new-run`.  It rebuilds if necessary.

Reply:
ok